intelmq/bots/outputs/rpz_file/output.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-
+
+"""
+RPZ file output
+
+SPDX-FileCopyrightText: 2021 Marius Karotkis <marius.karotkis@gmail.com>
+SPDX-License-Identifier: AGPL-3.0-or-later
+"""
+
+from datetime import datetime
+import os
+from collections import defaultdict
+from pathlib import Path
+
+from intelmq.lib.bot import OutputBot
+
+RPZ_INDICATOR_MAP = {
+    "source.fqdn": "Intel::DOMAIN"
+}
+
+now = datetime.now()  # for timestamp
+
+
+class RpzFileOutputBot(OutputBot):
+    _file = None
+    format_filename: bool = False
+    __Bot_is_multithreadable = False
```

Comment:
```suggestion
    _is_multithreadable = False
```
That was recently fixed/changed in 5c91a8c9b59541c147774a727d000d18c5dc1d2e you could not have foreseen this